src/screens/login/components/OidcOnboardingView.tsx

```diff
@@ -0,0 +1,102 @@
+import Minilog from '@cozy/minilog'
+import React, { useEffect, useState } from 'react'
+import { KeyboardAvoidingView, Platform, StyleSheet, View } from 'react-native'
+
+import { SupervisedWebView } from '/components/webviews/SupervisedWebView'
+
+import { parseOidcOnboardingFinishedUrl } from './functions/oidc'
+
+import { NetService } from '/libs/services/NetService'
+import { routes } from '/constants/routes'
+
+const log = Minilog('OidcOnboardingView')
+
+interface ClouderyWebViewProps {
+  code: string
+  onboardUrl: string
+  startOidcOAuth: (fqdn: string, code: string) => void
+}
+
+/**
+ * Display a WebView with specidied 'onboardUrl' and call startOidcOauth when
+ * navigation to OIDCInstanceCreationResult url occurs
+ */
+export const OidcOnboardingView = ({
+  code,
+  onboardUrl,
+  startOidcOAuth
+}: ClouderyWebViewProps): JSX.Element => {
+  const [loading, setLoading] = useState(true)
+  const [displayOverlay, setDisplayOverlay] = useState(true)
+
+  const handleNavigation = (request: WebViewNavigation): boolean => {
+    const createdInstance = parseOidcOnboardingFinishedUrl(request)
+    if (createdInstance) {
+      startOidcOAuth(createdInstance.fqdn, code)
+      return false
+    }
+
+    return true
+  }
+
+  useEffect(() => {
+    // Wait 1ms before hiding overlay to prevent screen flickering
+    if (!loading) {
+      setTimeout(() => {
+        setDisplayOverlay(false)
+      }, 1)
+    }
+  }, [loading])
+
+  const Wrapper = Platform.OS === 'ios' ? View : KeyboardAvoidingView
+
+  return (
+    <Wrapper style={styles.view} behavior="height">
+      <SupervisedWebView
+        source={{ uri: onboardUrl }}
+        onShouldStartLoadWithRequest={handleNavigation}
+        onLoadEnd={(): void => setLoading(false)}
+        style={{
+          backgroundColor: '#4b4b4b'
```

Comment:
theme/color?

Reply:
This was a temporary value and repalced in [feat: Adapt LoginScreen overlay and bar colors to Cloudery's background](https://github.com/cozy/cozy-flagship-app/pull/700/commits/ce05b46cbc939b974b0a22f3a1e9d6942ec54311) commit